src/view/channel-detail.js

```diff
@@ -49,9 +54,18 @@ const ChannelDetailView = ({ store, nav }) => (
         <DetailField name="Balance">
           {store.selectedChannel.localBalanceLabel} {store.unitLabel}
         </DetailField>
-        <Button style={styles.deleteBtn} onPress={() => nav.goChannelDelete()}>
-          <ButtonText style={styles.deleteBtnText}>CLOSE CHANNEL</ButtonText>
-        </Button>
+        {store.selectedChannel.active ? (
+          <Button
+            style={styles.deleteBtn}
+            onPress={() => nav.goChannelDelete()}
+          >
+            <ButtonText style={styles.deleteBtnText}>CLOSE CHANNEL</ButtonText>
+          </Button>
+        ) : (
+          <Button style={styles.deleteBtn} onPress={() => nav.goChannels()}>
+            <ButtonText style={styles.deleteBtnText}>DONE</ButtonText>
+          </Button>
+        )}
```

Comment:
Why do we need to display a `DONE` button when there is already an `X` at the top right of the modal?

Also, we should be aware that there are certain cases where the user may want to force close non-active channels. In any case this change doesn't belong in this PR IMHO.

Reply:
To clarify, if you look at the current implementation of `closeSelectedChannel` in the channels action, it allows force closing. This change would remove that feature. If we want to change the UX for force closing, we should leave that for another PR.

Reply:
Ah that's my mistake, last time I checked force closing was never enabled. Fixed!